docs/exploring-es-data.asciidoc

```diff
@@ -63,6 +63,10 @@ GET  /_template/your-template-name
 // CONSOLE
 
 You can read more about {ref}/indices-templates.html[Index Templates] and how they are used.
+
+Another option is to use the {kibana-ref}/managing-indices.html[Kibana Index Management UI]. 
```

Comment:
This could as well be left out, but up to you. Kibana is all about exploring ES data (not only managing indices), so this feels a bit unnecessary.

Reply:
Ofc Kibana is more, but in this section we describe how you can retrieve your APM related `settings` and `mappings`. And with the Kibana Index Management UI you now also can view it via this and not only by doing an API call.